actionmailer/lib/action_mailer/caching.rb

```diff
@@ -0,0 +1,22 @@
+require 'action_dispatch/caching'
+
+module ActionMailer
+  module Caching
```

Comment:
I think we can remove this module and just include its content directly in `ActionMailer::Base`